app/views/vacancies/_school_overview_section.html.haml

```diff
@@ -61,7 +61,7 @@
   - if @vacancy.school.geolocation
     %div{ id: 'map_zoom', role: 'presentation', aria: { hidden: 'true', label: t('schools.map_aria_label') },  }
     = render partial: '/vacancies/school', formats: [:js], locals: { name: @vacancy.school.name,
-                                                          lat: @vacancy.school_geolocation.x,
-                                                          lng: @vacancy.school_geolocation.y }
+                                                          lat: @vacancy.school_or_school_group_geolocation.x,
```

Comment:
I'm confused about if you are using methods that don't exist.

Reply:
Getting the locations for `SchoolGroups` isn't in the scope of this ticket as I read it?

Reply:
I am, too.

Reply:
Is this a misfiring replaceall?

Reply:
It's coming from a delegation on `Vacancy` line 231.  Personally, I'm not a fan of this much metaprogramming relational magic, but I'm happy that it addresses the need right now.

Reply:
Can you copy (or permalink) me the line on Vacancy please @tatyree as I can't see it and would like to understand what is creating this method.